@kiva/kv-components/vue/stories/KvTabs.stories.js

```diff
@@ -5,6 +5,9 @@ import KvTabPanel from '../KvTabPanel.vue';
 export default {
 	title: 'KvTabs',
 	component: KvTabs,
+	args: {
+		vertical: true,
```

Comment:
I'm guessing you don't want vertical for all of the stories. If so, add to the `VerticalOrientation` story below.

Reply:
Agree, please verify that behavior and appearance of the existing stories.

Reply:
Well, it looks like our other existing stories aren't "applying" the default setting to their props. Thus, there were no changes to these stories in the Chromatic diff.